client/post-editor/controller.js

```diff
@@ -131,10 +133,14 @@ module.exports = {
 			// We have everything we need to start loading the post for editing,
 			// so kick it off here to minimize time spent waiting for it to load
 			// in the view components
-			if ( postID ) {
+			if ( postID && ! postToCopyId ) {
```

Comment:
Instead of the `!` here and additional condition below, you could have `postToCopyId` first, deferring this to the next `else if` as just `else if ( postID ) {`